recipes/MDAnalysisTests/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "MDAnalysisTests" %}
+{% set version = "0.15.0" %}
+{% set sha256 = "a41bde911f4bc176fecd0af0947fde4916718d544f1556af72fc878ba6a92bde" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py3k or win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - mdanalysis =={{ version }}
+    - nose
+    - psutil
+    - mock
+
+test:
+  imports:
+    - MDAnalysisTests
+    - MDAnalysisTests.datafiles
+
+about:
+  home: https://github.com/MDAnalysis/mdanalysis
+  license: GPLv2
```

Comment:
I think we prefer `GPL-2.0`. See the following list for abbreviations:
https://opensource.org/licenses/alphabetical

I'm happy to merge after that change.